src/internal/webawesome-element.ts

```diff
@@ -159,6 +159,17 @@ export default class WebAwesomeElement extends LitElement {
     return this.hasStatesSupport() ? this.internals.states.has(state) : false;
   }
 
+  /**
+   * Given a native event, this function ensures it's composed and, if not, dispatches it again as a composed event.
+   * This is useful for relaying native events such as `change`, which will otherwise not be retargeted. It is safe,
+   * albeit sloppy, to call this on composed events, as it will no-op.
+   */
+  dispatchComposedEvent(event: Event) {
+    if (!event.composed) {
+      this.dispatchEvent(new Event(event.type, { ...event, composed: true }));
```

Comment:
This will cast everything to `Event`, should we do this instead?

```suggestion
      this.dispatchEvent(new event.constructor(event.type, { ...event, composed: true }));
```